gradle-plugin/README.md

```diff
@@ -32,15 +32,18 @@ plugins {
 }
 
 emerge {
-  apiToken.set(System.getenv("EMERGE_API_TOKEN"))
```

Comment:
nit: can we just update the comment to say something like "// we use `EMERGE_API_TOKEN` by default so no need to set that!"? It's convenient having a `System.getenv()` example to copy if they use a different name.

Reply:
Worst case they accidentally duplicate setting this value which isn't a big deal.

Reply:
Sure!